Source/Core/VideoCommon/CommandProcessor.cpp

```diff
@@ -91,21 +90,15 @@ void DoState(PointerWrap& p)
   p.Do(s_interrupt_waiting);
 }
 
-static inline void WriteLow(volatile u32& _reg, u16 lowbits)
+static inline void WriteLow(std::atomic<u32>& reg, u16 lowbits)
 {
-  Common::AtomicStore(_reg, (_reg & 0xFFFF0000) | lowbits);
+  reg.store((reg.load(std::memory_order_relaxed) & 0xFFFF0000) | lowbits,
```

Comment:
Just wondering, is this _still_ atomic in the end (with a seperate `load` then `store`)?
_Then again, thats what the old code did, so same issue there...?_

Reply:
This is indeed not atomic. I mentioned this in the PR description.

Reply:
Fair enough, I read that and dismissed it from my head while looking at `LowPart`/`HighPart` (only remembering the `Low`/`High` part).